open-apparel-registry/open-apparel-registry
# Add scripts/console

## Overview

One of the "scripts to rule them all" that we had not yet implemented on this project. One helpful use of this script (which prompted us to add it to the scripts folder) is logging into the running Django container and running `pip freeze` to check the version numbers of installed dependencies.

https://github.com/github/scripts-to-rule-them-all

Connects #1294

## Testing Instructions

* Run `./scripts/server`
* Run `./scripts/console app`. Use `ls` to verify that the cwd is the JS application.
* Run `./scripts/console django`. Use `ls` to verify that the cwd is the Django application.
* Run `./scripts/console nope` and verify that help is displayed.

## Checklist
